<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMasterViewControls</class>
 <widget class="QWidget" name="QmitkMasterViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_MainFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenDicomImport">
        <property name="text">
         <string>Dicom Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenPreprocessingReconstruction">
        <property name="text">
         <string>Preprocessing / Reconstruction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenQuantification">
        <property name="text">
         <string>Quantification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenTractography">
        <property name="text">
         <string>Tractography</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenTBSS">
        <property name="text">
         <string>TBSS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenConnectomics">
        <property name="text">
         <string>Connectomics</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenIVIM">
        <property name="text">
         <string>IVIM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenVolumeVisualization">
        <property name="text">
         <string>Volume Visualization</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_OpenScreenshotsMovies">
        <property name="text">
         <string>Screenshots / Movies</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
